src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,20 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_in_process(extra_env):
```

Comment:
I would have named this "use_radamsa_mutator_plugin" the current name might be confusing (in which process?).